src/common/vars/application-information-token.injectable.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+import { getInjectionToken } from "@ogre-tools/injectable";
+import type packageJson from "../../../package.json";
+
+export type ApplicationInformation = Pick<typeof packageJson, "version" | "config" | "productName" | "copyright" | "description" | "name"> & {
+  build: { publish?: unknown[] };
+};
+
+const applicationInformationToken = getInjectionToken<ApplicationInformation>({
```

Comment:
Note: token files don't need to be named `*.injectable.*`

Reply:
Good catch, will fix.